README.md

```diff
@@ -110,6 +110,7 @@ To switch your Front-End application to use Scalelite instead of a single BigBlu
 * `URL_HOST`: The hostname that the application API endpoint is accessible from. Used to protect against DNS rebinding attacks. Should be left blank if deploying Scalelite behind a Network Loadbalancer.
 * `SECRET_KEY_BASE`: A secret used internally by Rails. Should be unique per deployment. Generate with `bundle exec rake secret` or `openssl rand -hex 64`.
 * `LOADBALANCER_SECRET`: The shared secret that applications will use when calling BigBlueButton APIs on the load balancer. Generate with `openssl rand -hex 32`
+* `LOADBALANCER_SECRETS`: Additional shared secrets, separated by `:`. Any of these secrets will work. In an environment where multiple applications need to integrate with a single scalelite server, it may be sensible to give each application its own secret. This way, revoking individual secrets later will not disturb other applications. 
```

Comment:
Probably should separate them with a `,` and not `:`

Reply:
I opted for `:` because it is the default separator for other important environment variables (e.g. `PATH`) and `,` may imply that more sophisticated csv syntax is allowed (with whitespace stripping, quotes) which is not the case.